config/config.js

```diff
@@ -0,0 +1,25 @@
+require('dotenv').config()
+
+module.exports = {
+    development: {
+        username: process.env.POSTGRES_USER,
+        password: String(process.env.POSTGRES_PASSWORD),
```

Comment:
_(here and below)_
All the values inside `process.env` are always strings, so we could omit redundant `String(string)`.
```suggestion
        password: process.env.POSTGRES_PASSWORD,
```